src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,173 @@
+For more information about writing mobile applications using Dart,
+see [flutter.io]({{site.flutter}}).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Dart VM libraries:**
+If you write servers or command-line applications, you'll
+want to use [dart:io](https://api.dartlang.org/stable/dart-io/dart-io-library.html)
+and related libraries, such as
+[http_server](https://pub.dartlang.org/packages/http_server).
+For more information, see [Dart VM](/dart-vm/).
+</aside>
+
+## Dart libraries and their corresponding packages
+
+Dart's core libraries contain classes most commonly used by developers.
+Classes that are still useful, but not as universally used,
+are placed in packages outside of the SDK.
+Separating out the less commonly used classes ensures that your app
+isn't bloated by features that you don't need.
+For example, the
+[dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)
+library contains the Queue, HashMap, and HashSet classes. The
+[collection package](https://www.dartdocs.org/documentation/collection/latest/collection/collection-library.html)
+contains more specialized classes, such as CanonicalizedMap, MapKeySet, PriorityQueue,
+and support for equality testing.
+
+If you don't find the functionality you need on api.dartlang.org, check pub.dartlang.
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+Libraries and packages aren't interchangeable terms. Both libraries and
+applications are distributed as packages. A library can be part of a package,
+but a package is not a library.
+For more information on what constitutes a library package, see
+[Create Library Packages](/guides/libraries/create-library-packages).
+</aside>
+
+Some of the libraries in the SDK have corresponding packages with additional
+functionality. The following table includes some that you should know about:
+
+| **SDK Library** | **Package** |
+| [dart:async](https://api.dartlang.org/stable/dart-async/dart-async-library.html)<br>Support for asynchronous programming with classes such as Future and Stream. | [async](https://www.dartdocs.org/documentation/async/latest/)<br>Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html). |
+| [dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)<br>Classes and utilities that supplement the collection support in dart:core. | [collection](https://www.dartdocs.org/documentation/collection/latest)<br>Utility functions and classes to make working with collections easier. For more information, see [Unboxing Packages: collection](http://news.dartlang.org/2016/01/unboxing-packages-collection.html). |
```

Comment:
We need to surface class names here. I like that you're doing that for dart:async ("such as Future and Stream") but we need that everywhere. For SEO. Maybe a separate column with 'example members'?

Use Natalie's posts to get the really interesting member classes and functions.